smoke-test/tests/tokens/revokable_access_token_test.py

```diff
@@ -336,4 +414,52 @@ def loginAs(username, password):
     )
     response.raise_for_status()
 
-    return session
\ No newline at end of file
+    return session
+
+def removeUser(session, urn):
+    # Revoke token
+    json = {
+        "query": """mutation removeUser($urn: String!) {\n
+            removeUser(urn: $urn)
+        }""",
+        "variables": {
+          "urn": urn
+        }
+    }
+
+    response = session.post(
+        f"{FRONTEND_ENDPOINT}/api/v2/graphql", json=json
+    )
+    sleep(5)
```

Comment:
Why sleep 5 here?

Reply:
just to propagate?

Reply:
Yes, to ensure (as much as I can) that the message is processed by MCP processor